codegen/src/main/scala/akka/grpc/gen/scaladsl/play/PlayScalaServerCodeGenerator.scala

```diff
@@ -5,21 +5,23 @@
 package akka.grpc.gen.scaladsl.play
 
 import akka.grpc.gen.Logger
-import akka.grpc.gen.scaladsl.{ ScalaCodeGenerator, Service }
+import akka.grpc.gen.scaladsl.{ ScalaCodeGenerator, ScalaServerCodeGenerator, Service }
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorResponse
-import templates.PlayScala.txt.Router
+import templates.PlayScala.txt._
 
-object PlayScalaServerCodeGenerator extends PlayScalaServerCodeGenerator
-
-trait PlayScalaServerCodeGenerator extends ScalaCodeGenerator {
+case class PlayScalaServerCodeGenerator(powerApis: Boolean = false) extends ScalaCodeGenerator {
   override def name: String = "akka-grpc-play-server-scala"
 
-  override def perServiceContent = super.perServiceContent + generateRouter
+  override def perServiceContent = super.perServiceContent + generateRouter() ++ (
+    if (powerApis) Set(generateRouter(powerApis))
+    else Set.empty
```

Comment:
I generated both variants of the handler so that the plugin-tester tests could test both variants, so I figured we should be consistent and similarly generate both variants of the router.

But if you still think it preferable to only generate the power API router let me know and I'll make the change.

Reply:
Ah ok, let's keep it like this then